packages/sdk/src/api/GamesApi.ts

```diff
@@ -95,4 +103,12 @@ export class GamesApi {
       gameCode: this.options.gameId,
     });
   }
+
+  async getActiveTournamentData() {
+    const endpoint = this.createEndpoint(`tournament/active/${this.options.gameId}`);
+    const response = await fetch(endpoint);
+    const data: Tournament[] = await response.json();
```

Comment:
The shorter version might be this:

```suggestion
    const [tournament]: Tournament[] = await response.json();
```

and then just return `tournament`

Reply:
fixed